README.md

```diff
@@ -16,6 +16,7 @@ Brayns also comes with some already-made plugins:
 * **CircuitInfo (Deprecated)**: A plugin which accepts queries to extract information from neuronal circuits.
 * **DTI**: A diffuse-tensor imaging visualization plugin.
 * **AtlasExplorer**: A plugin capable of interpreting and render NRRD volumes.
+* **CylindricCamera**: A plugin that adds a cylindric camera projection for inmersive environments.
```

Comment:
I would have called it OpenDeckPlugin maybe so if we need to develop other things for the OpenDeck, we can put it here.

Reply:
Even though its true we are going to use it for the opendeck, I made the camera configurable so that it can be used for any cylindrical projection (configurable fovy), I didnt want to mark it as something specific for the opendeck.